app/src/main/java/org/mozilla/fenix/settings/CustomizationFragment.kt

```diff
@@ -142,11 +142,6 @@ class CustomizationFragment : PreferenceFragmentCompat() {
     }
 
     private fun setupGesturesCategory() {
-        requirePreference<SwitchPreference>(R.string.pref_key_website_pull_to_refresh).apply {
```

Comment:
I am wondering if we should remove this setting from the Customization Settings with this PR. Maybe we should file a separate issue for removing the setting to get UX to weigh in.

Reply:
Removing that setting follows UX recommendations from https://github.com/mozilla-mobile/fenix/issues/15629

Reply:
Would've been better to resolve remove this setting as part of that ticket but that refers to other changes also and this will probably be merged before the others, Just wanted to have it right from the start.

Reply:
Gotcha, I think it makes sense if we were enabling Pull to Refresh beyond Nightly, but since we are only enabling for Nightly, maybe we should keep it for the time being and have it serve as an easy debugging tool to see if problems with pull to refresh is due to the feature itself. We can keep the settings removal in a separate PR until we have a bit more QA time baked in.